node/node.d.ts

```diff
@@ -2244,102 +2244,108 @@ declare module "crypto" {
 declare module "stream" {
     import * as events from "events";
 
-    export class Stream extends events.EventEmitter {
+    class internal extends events.EventEmitter {
```

Comment:
How about this?
I want to avoid see confusing error message, like this `error TS2305: Module 'internal' has no exported member 'Stresam'.`.
use `_Stream` instead of internal? Do you have any good naming?